module-apps/apps-common/widgets/TimeFixedWidget.hpp

```diff
@@ -35,11 +41,22 @@ namespace gui
                         const std::uint32_t &h,
                         const bool minus = false);
 
+        TimeFixedWidget(Item *parent,
+                        const std::uint32_t &x,
+                        const std::uint32_t &y,
+                        const std::uint32_t &w,
+                        const std::uint32_t &h,
+                        const bool minus,
+                        std::uint32_t leftBoxSize,
+                        std::uint32_t rightBoxSize);
+
+        void init(const std::uint32_t &w, const std::uint32_t &h);
```

Comment:
Do we really want to pass `uint32_t`s by reference here? There is no difference in size in this case but it makes the code more complex than necessary.

Reply:
I will remove &